OrbitCore/SymbolHelper.cpp

```diff
@@ -86,8 +86,8 @@ outcome::result<ModuleSymbols, std::string> FindSymbols(
     return symbols_file->LoadSymbols();
   }
 
-  return absl::StrFormat("Could not find symbols for module \"%s\"",
-                         module_path);
+  return outcome::failure(
```

Comment:
Should we not completely abstract away outcome in our usage? That is, this should read as 

return Result(absl::StrFormat(...));

Reply:
Isn't it possible to use `ErrorMessage` directly in this case?

Reply:
It is possible to return ErrorMessage in this case, I find outcome::failure to be more readable.

we could make an alias for outcome::failure as well I guess.

Reply:
Having said that this can be resolved by explicitly calling has_failure() instead of implicitly converting to boolean.

Reply:
Yes it is possible. 
The style that I would prefer is using just the regular return type in case of success and `ErrorMessage` in case of failure.

`Result<void, ErrorMessage>` unfortunately requires to use `outcome::success()`.